src/Sylius/Bundle/ProductBundle/spec/EventListener/SelectProductAttributeChoiceRemoveListenerSpec.php

```diff
@@ -100,10 +107,8 @@ function it_does_not_remove_select_product_attribute_choices_if_there_is_only_ad
             ],
         ]);
 
-        $entityManager
-            ->getRepository('Sylius\Component\Product\Model\ProductAttributeValue')
-            ->shouldNotBeCalled()
-        ;
+        $entityManager->getUnitOfWork()->willReturn($unitOfWork);
+        $entityManager->getRepository('Sylius\Component\Product\Model\ProductAttributeValue')->shouldNotBeCalled();
```

Comment:
Is there any reason for not using `ProductAttributeValue::class`?

Reply:
Not really, just wanted not to change too much as I would rather rewrite the whole test instead :)